### PR TITLE
[Console] Add icons to logs

### PR DESCRIPTION
**Pull request by avanderhoorn**
screen shot 2017-06-26 at 9 10 02 am

### CODE REVIEW

File: src/views/logs.js [hunk 1]
```diff
@@ -2,6 +2,24 @@ const dom = require('../lib/dom');
 const consoleProxy = require('../proxy/console');
 const summaryRepository = require('../repository/summary');
 
+const icons = {
```
Comment (davidkpiano) [inline src/views/logs.js]: After my PR is pulled in, these can go in `assets/icons.js`

File: src/views/logs.js [hunk 2]
```diff
@@ -55,13 +73,17 @@ function updateSummary() {
...
-        + data.info;
+    const content = `
+        <span${iconSectionHasValue(data.error)}>${icons.error} ${data.error}</span>
+        <span${iconSectionHasValue(data.warn)}>${icons.warn} ${data.warn}</span>
+        <span${iconSectionHasValue(data.info)}>${icons.info} ${data.info}</span>
+    `;
 
     document.getElementById(target).innerHTML = content;
 }
+function iconSectionHasValue(total) {
```
Comment (davidkpiano) [inline src/views/logs.js]: Nit: rename to `iconSectionHasValueClass` or something like that... at first glance I expect a boolean from this function

File: src/views/logs.js [hunk 3]
```diff
@@ -2,6 +2,24 @@ const dom = require('../lib/dom');
...
+            <path d="M1024,0c565,0,1024,459,1024,1024S1589,2048,1024,2048,0,1589,0,1024,459,0,1024,0Zm113,1024,342-342L1366,569,1024,911,682,569,569,682l342,342L569,1366l113,113,342-342,342,342,113-113-342-342Z" />
+        </svg>
+    `,
+    warn: `
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2048 2048" class="icon-warn">
+            <path d="M1984,1984H64L1024,64l960,1920Zm-896-384.00006H960.00006V1728H1088V1599.99994ZM1088,1472V832H960.00006v640Z" />
+        </svg>
+    `,
+    info: `
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2048 2048" class="icon-info">
```
Comment (legomushroom) [inline src/views/logs.js]: Is there any reason we keep the `SVG` in `js` file?

File: src/index.scss [hunk 4]
```diff
@@ -452,3 +452,28 @@ $popup-delay: .3s;
     color: $color-primary !important;
     text-decoration: none !important;
 }
+
+.glimpse-logs {
+    svg {
+        height: 12px !important;
+        width: 12px !important;
+        fill: rgba(white, 0.5) !important;
```
Comment (legomushroom) [inline src/index.scss]: Just heads up, if we aim to support IE8, `rgba` colors and `SVG` is not supported in IE8.

**Review by legomushroom — COMMENTED** [review]
Few comments, lgtm

File: src/views/logs.js (hunk 1 above)
Comment (avanderhoorn) [reply src/views/logs.js]: Yep now shifted.

File: src/views/logs.js (hunk 2 above)
Comment (avanderhoorn) [reply src/views/logs.js]: No problem.

File: src/views/logs.js (hunk 3 above)
Comment (avanderhoorn) [reply src/views/logs.js]: Its the way @davidkpiano did it so being constant.

File: src/index.scss (hunk 4 above)
Comment (avanderhoorn) [reply src/index.scss]: IE8 support not needed. Hud has the same browser support as the client.

File: src/views/logs.js (hunk 3 above)
Comment (legomushroom) [reply src/views/logs.js]: @davidkpiano there are big benefits in keeping SVG in svg file, can we do that?

File: src/index.scss (hunk 4 above)
Comment (legomushroom) [reply src/index.scss]: I thought we want to keep HUD as widely supported as possible vs (2 last versions for CLient). Good to know.

**Comment by avanderhoorn** [comment]
Latest image:
screen shot 2017-06-26 at 9 19 36 am

File: src/views/logs.js (hunk 3 above)
Comment (davidkpiano) [reply src/views/logs.js]: @legomushroom Yes, soon!

**Pull request merged**